omero/developers/Python.rst

```diff
@@ -375,6 +431,28 @@ Read-Annotate groups, this will include other users' data - see
     image = conn.getObject("Image", imageId)
     print "Image: ", image,
 
+- **To save an Object's owner as another user (Admins only)**
+
+::
+
+    tag_ann = omero.gateway.TagAnnotationWrapper(conn)
+    tag_ann.setTextValue("Not owned by me")
+    # update details of the wrapped omero.model.AnnotationI _obj
+    tag_ann._obj.details.owner = ExperimenterI(userId, False)
+    tag_ann.save()
+
+    # Alternatively, we can connect as another user, to perform several tasks
```

Comment:
"Alternatively..." - I am not sure at this point what are we doing, what is the larger purpose of this effort ? This might even "become clear from the code", but still, I think it would be good to warn me and give me a hint if I should read this section at all